src/__tests__/faulty-keywords.test.js

```diff
@@ -1,27 +1,29 @@
-const {checkForFaultyFields} = require(`../faulty-keywords`);
+import {checkForFaultyFields} from '../faulty-keywords';
 
-const faultyResponse = {
-  allTracks: [
-    {length: 214},
-    {length: 184}
-  ]
-};
+describe('Faulty keywords tests', () => {
+  const faultyResponse = {
+    allTracks: [
+      {length: 214},
+      {length: 184}
+    ]
+  };
 
-const swellResponse = {
-  allTracks: [
-    {aliasedLength: 214},
-    {aliasedLength: 184}
-  ]
-};
+  const swellResponse = {
+    allTracks: [
+      {aliasedLength: 214},
+      {aliasedLength: 184}
+    ]
+  };
 
-it(`is truthy if the query contains a faulty keyword`, () => {
-  expect(
-    checkForFaultyFields(faultyResponse)
-  ).toBeTruthy();
-});
+  it(`detects faulty keywords`, () => {
+    expect(
+      checkForFaultyFields(faultyResponse)
+    ).toBeTruthy();
+  });
 
-it(`is falsy if the faulty keyword is aliased`, () => {
-  expect(
-    checkForFaultyFields(swellResponse)
-  ).toBeFalsy();
+  it(`ignores faulty keywords if they are aliased`, () => {
```

Comment:
Would even say this is not even the test case. It's more like... `detects when there is no faulty keywords` or something like that. Nothing in the code is actually checking if its a keyword which was aliased...